pypsa/plot.py

```diff
@@ -37,19 +37,9 @@
 __copyright__ = "Copyright 2015-2020 Tom Brown (FIAS), Jonas Hoersch (FIAS); Copyright 2019-2020 Fabian Hofmann (FIAS), Fabian Neumann (KIT), GNU GPL 3"
 
 
-plt_present = True
-try:
-    import matplotlib.pyplot as plt
-    from matplotlib.patches import Wedge
-    from matplotlib.collections import LineCollection, PatchCollection
-except ImportError:
-    plt_present = False
-
-basemap_present = True
-try:
-    from mpl_toolkits.basemap import Basemap
-except ImportError:
-    basemap_present = False
+import matplotlib.pyplot as plt
+from matplotlib.patches import Wedge
+from matplotlib.collections import LineCollection, PatchCollection
```

Comment:
`plt_present` removed because matplotlib is in dependencies I assume?

Reply:
Indeed